docs/tutorials/graphql-service.rst

```diff
@@ -78,7 +78,88 @@ The equivalent in our service is:
 Getting Points from a Spec
 --------------------------
 
-TODO
+Most importantly, is the ability to obtain a list of scan points from a Spec. 
+GraphQL gives the user the ability to request one or more fields from an object, 
+allowing them to obtain data that is relevant only to their application.
+
+The 'getPoints' query makes use of this, giving users the ability to select from
+one or more of the following fields:
+
+- numFrames: the total number of frames produced by the Spec
+- returnedFrames (WIP): the number of frames returned, limited by the maxPoint arguement
+- axes: a list of axes present in the Spec and its associated scan points
+
+Within axes:
+- axis: a list of the axes present in the Spec
```

Comment:
Isn't this the name of the axis being examined?

Reply:
Currently theres no way to filter by axis, so calling axis will always return all of the axes present in the scan. Similarly, lower, midpoints and upper will always return the points present in all axes.

Reply:
I meant that within axes, axis is the name of the axis present in the Spec, not a list

Reply:
ah I understand now